qt/MakeDialog.cc

```diff
@@ -220,24 +220,17 @@ void MakeDialog::onSourceChanged()
         builder_.emplace(filename.toStdString());
     }
 
-    QString text;
-
     if (!builder_)
     {
-        text = tr("<i>No source selected</i>");
+        QString text = tr("<i>No source selected</i>");
```

Comment:
Consider moving this text change to `updatePiecesLabel()` as well so that it's all in one place.

Reply:
I thought I had a good reason for not doing that, but it might have been before I broke things out into the separate call. :+1: